### PR TITLE
Dark mode for articles final

### PR DESCRIPTION
**Pull request by puneethkanna**

### CODE REVIEW

File: _includes/menu-header.html [hunk 1]
```diff
@@ -1,16 +1,13 @@
 <!--  Replace menu links here -->
 
 <li class="nav-item">
-<a class="nav-link" href="{{site.baseurl}}/index.html">Home</a>
+<a class="nav-link" href="/">Home</a>
```
Comment (SriSatyaLokesh) [inline _includes/menu-header.html:4]: is it working fine?

File: _layouts/post.html [hunk 2]
```diff
@@ -3,13 +3,15 @@
...
+<!--script src="https://cdn.jsdelivr.net/npm/clipboard@2/dist/clipboard.min.js"></script-->
+
 <!--Dark mode JS-->
 <script src="../assets/js/dark-mode.js"></script>
 <script>
   var options = {
-  bottom: '32px', // default: '32px'
-  right: 'unset', // default: '32px'
-  left: '32px', // default: 'unset'
+  top: '38px', // default: '32px'
```
Comment (SriSatyaLokesh) [inline _layouts/post.html:12]: top?

File: _includes/menu-header.html (hunk 1 above)
Comment (puneethkanna) [reply _includes/menu-header.html:4]: Will check it now.

File: _site/4-steps-you-should-take-to-secure-your-gmail-account/index.html [hunk 3]
```diff
@@ -238,9 +235,9 @@
     "body": "          Categorie :windows    {% assign category = site. posts | where: categories , windows  %}          {% for post in category %}      {% include main-loop-card. html %}      {% endfor %}                  {% include sidebar-featured. html %}            "
     }, {
     "id": 22,
-    "url": "http://localhost:4000/contact.html",
+    "url": "http://localhost:4000/contact",
     "title": "Contact",
-    "body": " If you’d like to get in touch with us, feel free to write us at devskrate@gmail. com: "
+    "body": " If you’d like to get in touch with us, feel free to write us at support@devskrate. com "
```
Comment (SriSatyaLokesh) [inline _site/4-steps-you-should-take-to-secure-your-gmail-account/index.html:240]: Who's maintaining this?

File: _includes/menu-header.html (hunk 1 above)
Comment (puneethkanna) [reply _includes/menu-header.html:4]: Yup, working.

File: _site/4-steps-you-should-take-to-secure-your-gmail-account/index.html [hunk 4]
```diff
@@ -133,9 +130,9 @@
     "body": " 404 Page not found :(  The requested page could not be found. "
     }, {
     "id": 1,
-    "url": "http://localhost:4000/about.html",
```
Comment (SriSatyaLokesh) [inline _site/4-steps-you-should-take-to-secure-your-gmail-account/index.html:136]: ?

File: _includes/menu-header.html (hunk 1 above)
Comment (puneethkanna) [reply _includes/menu-header.html:4]: > is it working fine?
Yup

File: _site/404.html [hunk 5]
```diff
@@ -269,151 +266,161 @@
...
     }, {
     "id": 29,
+    "url": "http://localhost:4000/page4/",
```
Comment (SriSatyaLokesh) [inline _site/404.html:269]: I'm not sure but why local host?

**Review by SriSatyaLokesh — APPROVED** [review]
there many many changes let's merge this to prod 
we will see in further patches

File: _layouts/post.html (hunk 2 above)
Comment (puneethkanna) [reply _layouts/post.html:12]: Opps.

File: _site/4-steps-you-should-take-to-secure-your-gmail-account/index.html (hunk 3 above)
Comment (puneethkanna) [reply _site/4-steps-you-should-take-to-secure-your-gmail-account/index.html:240]: Sometimes, jekyll is unable to regenearate the made changes in time.

Comment (puneethkanna) [reply _site/4-steps-you-should-take-to-secure-your-gmail-account/index.html:240]: The mistake is with the paths. Two files has same paths, so it took the first. It is fixed now.

File: _site/404.html (hunk 5 above)
Comment (puneethkanna) [reply _site/404.html:269]: As we run locally, they are created, but we will have no problem with it.

**Pull request merged**